pkg/http/handler/timeout.go

```diff
@@ -90,13 +92,32 @@ func (h *timeoutHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// the panic from h.handler.ServeHTTP if h.handler.ServeHTTP panics.
 	done := make(chan interface{})
 	tw := &timeoutWriter{w: w, clock: h.clock}
+
+	// make sure that when max duration time out expires
+	// curTime - requestTime >= timeout
+	tw.requestStartTime = tw.clock.Now()
```

Comment:
Probably reading the PR wrong, but I can't figure out where this field is used / what it does?

Reply:
Need to remove I guess now that I dont calculate any time diff.